add_pypi_package.sh

```diff
@@ -32,7 +33,13 @@ generate_pypi_package() {
   mkdir $PACKAGE_DIR
   echo "FINISHED"
   echo -n "Creating specs and downloading sources..."
-  pyp2rpm --no-autonc -s -d $PACKAGE_DIR -v $VERSION $PYPI_NAME
+  # pass -r $PACKAGE_NAME to pyp2rpm if we had to downcase the name
+  if [[ $PACKAGE_NAME != "python-${PYPI_NAME}" ]]; then
```

Comment:
Any reason to not always pass it? Doesn't hurt to be explicit :)

Reply:
The generated spec file is slightly noisier with it passed.

Reply:
example

```diff
diff --git a/obal.spec b/obal.spec
index cf31b7f..d517d96 100644
--- a/obal.spec
+++ b/obal.spec
@@ -1,7 +1,8 @@
 # Created by pyp2rpm-3.3.2
 %global pypi_name obal
+%global srcname obal
 
-Name:           python-%{pypi_name}
+Name:           python-%{srcname}
 Version:        0.5.1
 Release:        1%{?dist}
 Summary:        packaging wrapper using ansible
@@ -22,14 +23,14 @@ should also work with plain Ansible when called like ansible-playbook
 <action_playbook>.yml -l <package> instead of obal <action> <package>.
 necessary tools- python (2 or...
 
-%package -n     python3-%{pypi_name}
+%package -n     python3-%{srcname}
 Summary:        %{summary}
-%{?python_provide:%python_provide python3-%{pypi_name}}
+%{?python_provide:%python_provide python3-%{srcname}}
  
 Requires:       python3dist(ansible) >= 2.5
 Requires:       python3dist(argcomplete)
 Requires:       python3dist(setuptools)
-%description -n python3-%{pypi_name}
+%description -n python3-%{srcname}
  obal - packaging wrapper using Ansible[![Documentation Status]( is an Ansible
 wrapper with a set of Ansible playbooks to ease maintanance of packaging
 repositories like [foreman-packaging]( and [pulp-packaging]( obal actions
@@ -49,7 +50,7 @@ rm -rf %{pypi_name}.egg-info
 %install
 %py3_install
 
-%files -n python3-%{pypi_name}
+%files -n python3-%{srcname}
 %license LICENSE
 %doc README.md
 %{_bindir}/obal
```